### ISSUE TITLE
An error occurred while Bundler/DuplicatedGroup cop was inspecting /path/to/Gemfile.

### ISSUE THREAD

**Issue by MrJoy** [rubocop/rubocop#12123]
## Expected behavior

No error.

## Actual behavior

```
 % rubocop -d Gemfile
For /Users/jonathonfrisby/fb/core: configuration from /Users/jonathonfrisby/fb/core/.rubocop.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-capybara-2.18.0/config/default.yml
Default configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-capybara-2.18.0/lib/../config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-eightyfourcodes-0.0.2/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-eightyfourcodes-0.0.2/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-factory_bot-2.23.1/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-factory_bot-2.23.1/lib/../config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-graphql-1.4.0/config/default.yml
configuration from #{path}
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-performance-1.19.0/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-performance-1.19.0/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-rails-2.20.2/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-rails-2.20.2/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-rake-0.6.0/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-rake-0.6.0/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-rspec-2.23.2/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-rspec-2.23.2/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-rubycw-0.1.6/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-rubycw-0.1.6/config/default.yml
configuration from /Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-thread_safety-0.5.1/config/default.yml
configuration from #{path}
Use parallel by default.
Skipping parallel inspection: only a single file needs inspection
Inspecting 1 file
Scanning /Users/jonathonfrisby/fb/core/Gemfile
An error occurred while Bundler/DuplicatedGroup cop was inspecting /Users/jonathonfrisby/fb/core/Gemfile.
undefined method `value' for s(:splat,
  s(:const, nil, :LIVE_ENVS)):RuboCop::AST::Node
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/bundler/duplicated_group.rb:63:in `map'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/bundler/duplicated_group.rb:63:in `block in duplicated_group_nodes'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/bundler/duplicated_group.rb:69:in `block in group_declarations'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:111:in `block in visit_descendants'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:108:in `each'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:108:in `visit_descendants'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:112:in `block in visit_descendants'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:108:in `each'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:108:in `visit_descendants'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-ast-1.29.0/lib/rubocop/ast/node/mixin/descendence.rb:100:in `each_node'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/bundler/duplicated_group.rb:63:in `group_declarations'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/bundler/duplicated_group.rb:63:in `each'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/bundler/duplicated_group.rb:63:in `group_by'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/bundler/duplicated_group.rb:63:in `duplicated_group_nodes'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/bundler/duplicated_group.rb:49:in `on_new_investigation'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/commissioner.rb:160:in `block (2 levels) in invoke'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/commissioner.rb:171:in `with_cop_error_handling'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/commissioner.rb:160:in `block in invoke'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/commissioner.rb:160:in `each'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/commissioner.rb:160:in `invoke'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/commissioner.rb:84:in `investigate'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/team.rb:156:in `investigate_partial'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cop/team.rb:98:in `investigate'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:345:in `block in inspect_file'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:344:in `each'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:344:in `flat_map'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:344:in `inspect_file'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:287:in `block in do_inspection_loop'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:321:in `block in iterate_until_no_changes'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:314:in `loop'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:314:in `iterate_until_no_changes'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:283:in `do_inspection_loop'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:164:in `block in file_offenses'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:189:in `file_offense_cache'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:163:in `file_offenses'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:154:in `process_file'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:135:in `block in each_inspected_file'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:134:in `each'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:134:in `reduce'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:134:in `each_inspected_file'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:120:in `inspect_files'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/runner.rb:73:in `run'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cli/command/execute_runner.rb:26:in `block in execute_runner'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cli/command/execute_runner.rb:52:in `with_redirect'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cli/command/execute_runner.rb:25:in `execute_runner'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cli/command/execute_runner.rb:17:in `run'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cli/command.rb:11:in `run'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cli/environment.rb:18:in `run'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cli.rb:118:in `run_command'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cli.rb:125:in `execute_runners'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cli.rb:51:in `block in run'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cli.rb:77:in `profile_if_needed'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/lib/rubocop/cli.rb:43:in `run'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/exe/rubocop:19:in `block in <top (required)>'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/lib/ruby/3.2.0/benchmark.rb:311:in `realtime'
/Users/jonathonfrisby/.rbenv/versions/3.2.2/gemsets/fasterbetter-core/gems/rubocop-1.56.0/exe/rubocop:19:in `<top (required)>'
/Users/jonathonfrisby/fb/core/bin/rubocop:29:in `load'
/Users/jonathonfrisby/fb/core/bin/rubocop:29:in `<main>'
.

1 file inspected, no offenses detected

1 error occurred:
An error occurred while Bundler/DuplicatedGroup cop was inspecting /Users/jonathonfrisby/fb/core/Gemfile.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop/rubocop/issues

Mention the following information in the issue report:
1.56.0 (using Parser 3.2.2.3, rubocop-ast 1.29.0, running on ruby 3.2.2) [arm64-darwin21]
Finished in 0.4647229998372495 seconds
```

## Steps to reproduce the problem

Gemfile attached.

However, this appears to specifically be caused by this construction:

```ruby
group(*LIVE_ENVS) do
  gem "cloudflare-rails", "~> 4.0.0"
  gem "sentry-rails",     "~> 5.0"
  gem "sentry-ruby",      "~> 5.0"
  gem "sentry-sidekiq",   "~> 5.0"
end
```

## RuboCop version

```
1.56.0 (using Parser 3.2.2.3, rubocop-ast 1.29.0, running on ruby 3.2.2) [arm64-darwin21]
  - rubocop-capybara 2.18.0
  - rubocop-factory_bot 2.23.1
  - rubocop-graphql 1.4.0
  - rubocop-performance 1.19.0
  - rubocop-rails 2.20.2
  - rubocop-rake 0.6.0
  - rubocop-rspec 2.23.2
  - rubocop-rubycw 0.1.6
  - rubocop-thread_safety 0.5.1
```
[Gemfile.txt](https://github.com/rubocop/rubocop/files/12354055/Gemfile.txt)

**Comment by koic** [comment]
This issue is a dup with #12111 and resolved by #12112. Thank you.
